Tests/test_image.py

```diff
@@ -139,7 +139,6 @@ def test_comparison_with_other_type(self):
         # Act/Assert
         # Shouldn't cause AttributeError (#774)
         self.assertFalse(item is None)
-        self.assertFalse(item == None)
```

Comment:
This is actually testing the `__eq__` method, whereas `is` does not.

```
class C:
    def __eq__(self, other):
        print("method tested")


a = C()
b = C()
print("is check")
a is b
print("equal check")
a == 0
```

Instead of deleting the line, my suggested fix would be to change it to testing `== 0`. This calls `__eq__`, without triggering flake8.

Reply:
Oh, wait, that's exactly what the next line does. Never mind then.

Reply:
Thanks for checking :)